tests/integration/resources/messages/message_pt_BR.properties

```diff
@@ -1,2 +1,3 @@
+[MESSAGES]
```

Comment:
@paulodamaso why is ``[MESSAGES]`` needed here? :D

Reply:
@amihaiemil I've decided to use python's `ConfigParser` lib, and it demands a title for each session